### DISCUSSION TITLE
More info about the `[image]' element on the about page.

### DISCUSSION THREAD

**Discussion by anoduck** [thuliteio/doks#1280] [Q&A] [answered]
The about page is a two column page with text content justified to the left and located in the left column, in the right column there is an element that simply reads `[image]`. 

I don't mind breaking out the editor and creating a custom layout for that page in order to obtain access to that element, but I am unsure if that is what it is for. Or is there some front matter label that allow the processing of the element into a picture.

**Comment by h-enk** [comment, marked as answer]
It's just a filler element — doing nothing but indicating that you could insert an image here (using one of the [image partials](https://images.thulite.io/docs/partials/))

Customize the layout in  `layouts/about/single.html` (copied from `node_modules/@thulite/doks-core/layouts/about/single.html`)

**Reply by anoduck** [reply]
Sweet design.
